src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -35,14 +35,17 @@ public function __construct()
      */
     public function find($includeArgs = true)
     {
+        $args = $this->findArguments();
+        $args = $includeArgs && $args ? ' '.implode(' ', $args) : '';
```

Comment:
Could you then at least split this in two separate commits? Otherwise it's really hard to use Git's blame feature to find the reason for a specific change in the future.